### ISSUE TITLE
publish a 1.5?

### ISSUE THREAD

**Issue by NorthIsUp** [kennknowles/python-jsonpath-rw#98]
lots of updates! would love to have them available on pip :)

**Comment by ailjushkin** [comment]
Yeah man, of course. But if you are worrying about possible crashes, maybe it's time to pre version, thus people could test it for some time, and then move to release?

**Comment by NorthIsUp** [comment]
Isn't that what version pinning is for?

**Comment by kennknowles** [comment]
I don't have much time to maintain this or do releases, but I am all about adding new collaborators.
